### PR TITLE
Add sysbench result for v1.0.0

### PR DESCRIPTION
**Pull request by lilin90**
PTAL

### CODE REVIEW

File: benchmark/sysbench.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+title: TiDB sysbench performance test report - v1.0.0
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Performace test result for TiDB using Sysbench

File: benchmark/sysbench.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+category: benchmark 
+---
+
+# TiDB sysbench performance test report - v1.0.0
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

File: benchmark/sysbench.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+## Test purpose
+
+The purpose of this test is to test the performance and horizontal scalability of TiDB in OLTP scenarios.
+
+## Test version, time and place
+
+TiDB version: v1.0.0
+
+Time: October 20, 2017
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Time -> Date

File: benchmark/sysbench.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+title: TiDB sysbench performance test report - v1.0.0
+category: benchmark 
+---
+
+# TiDB sysbench performance test report - v1.0.0
+
+## Test purpose
+
+The purpose of this test is to test the performance and horizontal scalability of TiDB in OLTP scenarios.
+
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Please add a statement:
> **Note**:  The results of the testing might vary based on different environmental dependencies.

File: benchmark/sysbench.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+Place: Beijing
+
+## Test environment
+
+IDC machines:
+
+| Category  |  Name       |  
+| :--------| :---------| 
+| OS       | linux (CentOS 7.3.1611)       |
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: linux -> Linux

File: benchmark/sysbench.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+## Test version, time and place
+
+TiDB version: v1.0.0
+
+Time: October 20, 2017
+
+Place: Beijing
+
+## Test environment
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Please add bullet list tag to the content in this section.

File: benchmark/sysbench.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+| Category  |  Name       |  
+| :--------| :---------| 
+| OS       | linux (CentOS 7.3.1611)       |   
+| CPU | 40 vCPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz |
+| RAM | 128GB | 
+| DISK | 1.5T SSD * 2  + Optane SSD * 1 |  
+
+Sysbench version: 1.0.6
+
+See the [test script](https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench).
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Test script: [https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench](https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench).

File: benchmark/sysbench.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+| RAM | 128GB | 
+| DISK | 1.5T SSD * 2  + Optane SSD * 1 |  
+
+Sysbench version: 1.0.6
+
+See the [test script](https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench).   
+
+## Test scenarios
+
+### Scenario one: sysbench standard performance test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: sysbench -> Sysbench

File: benchmark/sysbench.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+
+Test table structure:
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: The structure of the table used for the test:

File: benchmark/sysbench.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+
+Time: October 20, 2017
+
+Place: Beijing
+
+## Test environment
+
+IDC machines:
+
+| Category  |  Name       |
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Name -> Detail

File: benchmark/sysbench.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+
+See the [test script](https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench).   
+
+## Test scenarios
+
+### Scenario one: sysbench standard performance test
+
+Test table structure:
+
+```
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ->```sql

File: benchmark/sysbench.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `k` int(10) unsigned NOT NULL DEFAULT '0',
+  `c` char(120) NOT NULL DEFAULT '',
+  `pad` char(60) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`),
+  KEY `k_1` (`k`)
+) ENGINE=InnoDB  
+``` 
+
+Deployment and parameters configuration:
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: -> The deployment and configuration details:

File: benchmark/sysbench.md [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+172.16.20.6    4*tikv    1*tidb    1*sysbench
+172.16.20.7    4*tikv    1*tidb    1*sysbench
+172.16.10.8    1*tidb    1*pd      1*sysbench
+
+// Each physical node has three disks.
+data3: 2 tikv (Optane SSD) 
+data2: 1 tikv
+data1: 1 tikv
+
+// TiKV parameters configuration
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: TiKV parameters configuration ->TiKV configuration

File: benchmark/sysbench.md [hunk 14]
```diff
@@ -0,0 +1,207 @@
...
+
+// MySQL deployment
+// Use the semi-synchronous replication and asynchronous replication to deploy two replicas respectively.
+172.16.20.4    master    
+172.16.20.6    slave        
+172.16.20.7    slave
+172.16.10.8    1*sysbench 
+Mysql version: 5.6.37
+
+// MySQL parameters configuration
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: MySQL parameters configuration -> MySQL configuration

File: benchmark/sysbench.md [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+thread_cache_size = 64
+innodb_buffer_pool_size = 64G
+innodb_file_per_table = 1
+innodb_flush_log_at_trx_commit = 0  
+datadir = /data3/mysql  
+max_connections = 2000 
+```
+
+- Standard OLTP test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Standard -> Simple

File: benchmark/sysbench.md [hunk 16]
```diff
@@ -0,0 +1,207 @@
...
+    | Mysql | 32 | 1 million | 128 | 2493 | 49874 | 51.32 ms / 173.58 ms  |
+    | Mysql | 32 | 1 million | 256 | 2561 | 51221 | 99.95 ms  / 287.38 ms |
+    | Mysql | 32 | 5 million | 256 | 1902 | 38045 | 134.56 ms / 363.18 ms |
+    | Mysql | 32 | 10 million | 256 | 1770 | 35416 | 144.55 ms / 383.33 ms  |
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/threads_oltp.png?imageView2/2/w/700/q/75|imageslim)    
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/table_size_oltp.png?imageView2/2/w/700/q/75|imageslim)
+
+- Standard select test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Standard select test -> Simple `Select` test

File: benchmark/sysbench.md [hunk 17]
```diff
@@ -0,0 +1,207 @@
...
+    | Mysql | 32 | 1 million | 128 |  410426  |0.31 ms / 0.74 ms  |
+    | Mysql | 32 | 1 million | 256 |  396867 | 0.64 ms / 1.58 ms  |
+    | Mysql | 32 | 5 million | 256 |  386866 | 0.66 ms / 1.64 ms |
+    | Mysql | 32 | 10 million | 256 |  388273 | 0.66 ms / 1.64 ms  |
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/threads_select.png?imageView2/2/w/700/q/75|imageslim) 
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/table_size_select.png?imageView2/2/w/700/q/75|imageslim) 
+
+- Standard insert test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Simple `Insert` test

File: benchmark/sysbench.md [hunk 18]
```diff
@@ -0,0 +1,207 @@
...
+
+    | - | table count | table size | sysbench threads | qps | latency(avg / .95) |
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: Please keep the style of the header consistent.

File: benchmark/sysbench.md [hunk 19]
```diff
@@ -0,0 +1,207 @@
...
+    | Mysql | 32 | 5 million | 256 | 10593 | 24.16 ms / 82.96 ms  |
+    | Mysql | 32 | 10 million | 256 | 9813 | 26.08 ms / 94.10 ms  |  
+ 
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/threads_insert.png?imageView2/2/w/700/q/75|imageslim)
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/table_size_insert.png?imageView2/2/w/700/q/75|imageslim)   
+
+### Scenario two: TiDB horizontal scalability test
+
+Deployment and parameters configuration:
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

File: benchmark/sysbench.md [hunk 20]
```diff
@@ -0,0 +1,207 @@
...
+grpc-raft-conn-num = 24 
+[defaultcf]
+block-cache-size = "12GB"
+[writecf]
+block-cache-size = "5GB"
+[raftdb.defaultcf]
+block-cache-size = "2GB"
+``` 
+
+- Standard OLTP test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

File: benchmark/sysbench.md [hunk 21]
```diff
@@ -0,0 +1,207 @@
...
+    | - | Table count | Table size | Sysbench threads | TPS | QPS | latency(avg / .95) | 
+    | :---: | :---: | :---: | :---: | :---: | :---: | :---: | 
+    | 1 TiDB physical node | 32 | 1 million | 256 * 1 | 2495 | 49902 | 102.42 ms / 125.52 ms |
+    | 2 TiDB physical nodes | 32 | 1 million | 256 * 2 | 5007 | 100153 | 102.23 ms / 125.52 ms  |
+    | 4 TiDB physical nodes | 32 | 1 million | 256 * 4 | 8984 | 179692 | 114.96 ms / 176.73 ms |
+    | 6 TiDB physical nodes | 32 | 5 million | 256 * 6 | 12953 | 259072 | 117.80 ms / 200.47 ms  |
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/scale_tidb_oltp.png?imageView2/2/w/700/q/75|imageslim)
+
+- Standard select test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

File: benchmark/sysbench.md [hunk 22]
```diff
@@ -0,0 +1,207 @@
...
+    | - | table count | table size | sysbench threads | qps | latency(avg / .95) | 
+    | :---: | :---: | :---: | :---: | :---: | :---: | 
+    | 1 TiDB physical node | 32 | 1 million | 256 * 1 | 71841 | 3.56 ms / 8.74 ms |
+    | 2 TiDB physical nodes | 32 | 1 million | 256 * 2 | 146615 | 3.49 ms / 8.74 ms |
+    | 4 TiDB physical nodes | 32 | 1 million | 256 * 4 | 289933 | 3.53 ms / 8.74 ms  |
+    | 6 TiDB physical nodes | 32 | 5 million | 256 * 6 | 435313 | 3.55 ms / 9.17 ms  |
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/scale_tidb_select.png?imageView2/2/w/700/q/75|imageslim)
+
+- Standard insert test
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

File: benchmark/sysbench.md [hunk 23]
```diff
@@ -0,0 +1,207 @@
...
+    | 1 TiDB physical node | 32 | 1 million | 256 * 1 | 2495 | 49902 | 102.42 ms / 125.52 ms |
+    | 2 TiDB physical nodes | 32 | 1 million | 256 * 2 | 5007 | 100153 | 102.23 ms / 125.52 ms  |
+    | 4 TiDB physical nodes | 32 | 1 million | 256 * 4 | 8984 | 179692 | 114.96 ms / 176.73 ms |
+    | 6 TiDB physical nodes | 32 | 5 million | 256 * 6 | 12953 | 259072 | 117.80 ms / 200.47 ms  |
+
+    ![](http://7xnp02.com1.z0.glb.clouddn.com/scale_tidb_oltp.png?imageView2/2/w/700/q/75|imageslim)
+
+- Standard select test
+
+    | - | table count | table size | sysbench threads | qps | latency(avg / .95) |
```
Comment (queenypingcap) [inline benchmark/sysbench.md]: ditto

**Comment by queenypingcap** [comment]
@cwen0 @shenli PTAL.

File: benchmark/sysbench.md [hunk 24]
```diff
@@ -0,0 +1,209 @@
...
+// MySQL configuration
+thread_cache_size = 64
+innodb_buffer_pool_size = 64G
+innodb_file_per_table = 1
+innodb_flush_log_at_trx_commit = 0  
+datadir = /data3/mysql  
+max_connections = 2000 
+```
+
+- Simple OLTP test
```
Comment (cwen0) [inline benchmark/sysbench.md]: How about `OLTP RW Test`?  I don't think `Simple` is suitable.

Comment (cwen0) [reply benchmark/sysbench.md]: you can refer to this [Percona Database Performance Blog](https://www.percona.com/blog/2017/01/06/millions-queries-per-second-postgresql-and-mysql-peaceful-battle-at-modern-demanding-workloads/)

**Pull request merged**